src/views/InteractionDesigner.vue

```diff
@@ -208,8 +215,28 @@ export default {
     this.deselectBlocks()
     this.discoverTallestBlockForDesignerWorkspaceHeight({ aboveTallest: true })
 
+    setTimeout(() => {
+      const { blockId, field } = this.$route.params
+      if (blockId) {
+        this.activateBlock({ blockId })
+        const blockElement = document.querySelector(`#block\\/${blockId} .plain-draggable`)
+        if (blockElement) {
+          blockElement.scrollIntoView({ behavior: 'smooth', block: 'center' })
+        }
+      }
+      if (field) {
+        const ele = document.getElementById(`${blockId}.${field}`)
+        if (ele) {
+          ele.scrollIntoView({ behavior: 'smooth', block: 'center' })
+        }
+      }
```

Comment:
@poojahpatil90 Maybe we could consolidate this `setTimeout()` handler with our scrollBehaviour defined in routes/index.ts

Reply:
Done!